### DISCUSSION TITLE
Left panel auto zoom

### DISCUSSION THREAD

**Discussion by SiriusXT** [TriliumNext/Trilium#3621] [Show and tell]
When the tree on the left panel has a lot of long titles, it is not convenient to view. This script can automatically increase the width of the left panel when the mouse moves to the left.
https://github.com/SiriusXT/trilium-left-panel-auto-zoom

**Comment by SiriusXT** [comment]
> When the tree on the left panel has a lot of long titles, it is not convenient to view. This script can automatically increase the width of the left panel when the mouse moves to the left. https://github.com/SiriusXT/trilium-left-panel-auto-zoom

## Features
Three modes are supported :
    1. Automatically zoom in on the left panel when the mouse moves to the navigation bar. This is the default mode.
    2. Automatically zoom in on the left panel when the mouse moves to the left panel.
    3. Automatically zoom in on the left panel when the shortcut key alt+s is pressed.
## Preview

![preview](https://user-images.githubusercontent.com/37627919/222315848-676a7320-e3d4-4c59-a59d-66381a2d19b2.gif)

**Reply by zadam** [reply]
That looks pretty cool!

**Comment by Nriver** [comment]
The latest version works great now! The experience is very good, you should try it.
